src/main/resources/application.yaml

```diff
@@ -89,7 +89,7 @@ hapi:
     #    default_pretty_print: true
     #    default_page_size: 20
     #    delete_expunge_enabled: true
-    #    enable_repository_validating_interceptor: false
+    enable_repository_validating_interceptor: true
```

Comment:
Question (non-blocking): Did you intend to set this to true during a refactor? Seems out of place

Reply:
agree, this should be false by default

Reply:
That was a bug - only enabled for testing